### PR TITLE
IT-5127 drawer header

### PR DESCRIPTION
**Pull request by pioziol123**

### CODE REVIEW

File: lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+			white-space: nowrap;
+		}
+	}
+
+	&__titleWrapper {
+		align-items: center;
+		column-gap: $space-xxxxs;
+		display: flex;
+		justify-content: space-between;
+		min-height: 82px;
```
Comment (tweetgeek) [inline lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue:111]: why 82? we don't get this height from paddings?

File: lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+		},
+		isInteractiveEyebrow: {
+			type: Boolean,
+			default: false,
+		},
+		hasDivider: {
+			type: Boolean,
+			default: false,
+		},
+		eyebrowEllipsis: {
```
Comment (tweetgeek) [inline lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue:170]: i don't se this props in [Miro](https://miro.com/app/board/o9J_knPnexA=/?moveToWidget=3458764551320412081&cot=14), it is new? should we update it?

File: lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+						:icon="leftIcon"
+						:size="ICON_SIZES.X_SMALL"
+						class="drawerHeader__leftIcon"
+					/>
+					<span v-if="title" :class="{ '-ellipsis': titleEllipsis }">{{ title }}</span>
+					<pill v-if="pillLabel" :label="pillLabel" />
+				</div>
+			</div>
+			<icon-button
+				v-if="isClosable"
```
Comment (tweetgeek) [inline lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue:36]: is something wrong with size, based on figma it should be 32x32px now is 40x40px - so the whole component has incorrect height

Comment (rogatty) [reply lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue:36]: Probably we need:
```suggestion
				v-if="isClosable"
				:touchable="false"
```

File: lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue (hunk 2 above)
Comment (pioziol123) [reply lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue:170]: https://www.figma.com/file/oqNhXXGdc4ZnNQ6YNpkLEK?node-id=1823:641360#431804422

File: lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue [hunk 4]
```diff
@@ -21,7 +21,7 @@
 				>
 					{{ eyebrowText }}
 				</span>
-				<div class="drawerHeader__title">
+				<div class="drawerHeader__title" :title="title">
```
[the remark this answers is not in the document]
Comment (tweetgeek) [reply lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue:24]: look at https://github.com/bethinkpl/design-system/blob/e20b8be165305de109e142d3168f95581de13fa1/lib/js/components/ProgressBar/ProgressBar.vue#L15

**Pull request merged**